yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -751,7 +753,7 @@ def add(meta_list, info_list=None):
         # https://kodi.wiki/view/Video_file_tagging
 
         add('title', ('track', 'title'))
-        add('date', 'upload_date')
+        add('date', ('release_date', 'upload_date'), lambda date: strftime_or_none(date, '%Y-%m-%d'))
```

Comment:
Hmm, looks like we forgot `strftime_or_none` when we were adding `partial_application`. Let's do that:

```diff
diff --git a/yt_dlp/utils/_utils.py b/yt_dlp/utils/_utils.py
index a71a381e5..254eca39d 100644
--- a/yt_dlp/utils/_utils.py
+++ b/yt_dlp/utils/_utils.py
@@ -2046,6 +2046,7 @@ def url_or_none(url):
     return url if re.match(r'(?:(?:https?|rt(?:m(?:pt?[es]?|fp)|sp[su]?)|mms|ftps?):)?//', url) else None
 
 
+@partial_application
 def strftime_or_none(timestamp, date_format='%Y%m%d', default=None):
     datetime_object = None
     try:
```

and then:

```suggestion
        add('date', ('release_date', 'upload_date'), convert=strftime_or_none(date_format='%Y-%m-%d'))
```